### PR TITLE
Move strip gps information setting from blog settings to account settings.

### LINKED ISSUE #4850: Move option to remove the GPS information from photos to account level.
The option to strip GPS information from photos will be at an account level and by default it will be on in order to avoid privacy issues.

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fixes #4850

To test:
 - Go to account settings and check if new option to strip GPS location is there
 - Create a new post, add image with GPS information (make sure you the photos you use has location information)
 - See if the options has the desired effect of keeping and removing GPS information by downloading the image from  the server and checking the properties on Preview.

Needs review: @koke

### CODE REVIEW

File: WordPress/Classes/Services/MediaService.m [hunk 1]
```diff
@@ -53,10 +53,11 @@ - (void)createMediaWithPHAsset:(PHAsset *)asset
         allowedFileTypes = nil;
         mediaType = MediaTypeVideo;
     }
+
+    MediaSettings *mediaSettings = [MediaSettings new];
+    BOOL stripGeoLocation = !mediaSettings.removeLocationSetting;
```
Comment (koke) [inline WordPress/Classes/Services/MediaService.m]: I don't think this should be negated?

**Comment by koke** [comment]
Other than the extra `!`, code looks good. I'll test it tomorrow

**Comment by SergioEstevao** [comment]
@koke ready for another look.

**Comment by koke** [comment]
:shipit:

**Pull request merged**